html/pfappserver/root/configurator/configuration.tt

```diff
@@ -53,6 +53,20 @@
                 <p class="help-block">[% l('Email address to which notifications for rogue DHCP servers, violations with an action of <i>email</i>, or any other PacketFence-related messages goes to.') %]</p>
               </div>
             </div>
+            <h3>[% l('Database Passwords') %]</h3>
```

Comment:
Not sure it's any less ambiguous.
To an uneducated user (which is most people when they use the configurator) it is not much help.
Local database passwords?